src/corehost/cli/comhost/clsidmap.cpp

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "comhost.h"
+#include <trace.h>
+
+#include <cpprest/json.h>
+using namespace web;
+
+using comhost::clsid_map_entry;
+using comhost::clsid_map;
+
+#ifdef _WIN32
+bool GetModuleFileNameWrapper(HMODULE hModule, pal::string_t* recv);
+#endif
+
+namespace
+{
+    pal::dll_t get_current_module()
+    {
+#ifdef _WIN32
```

Comment:
These `#ifdefs` should not be necessary since this is for Windows only.

If it is ever ported to non-Windows, we can add them at that time.

Reply:
Sounds good.